src/components/v1/V1Project/modals/ProjectToolDrawerModal.tsx

```diff
@@ -76,6 +83,40 @@ export default function ProjectToolDrawerModal({
     )
   }
 
+  async function setArchived(archived: boolean) {
+    const newMetadata = {
+      ...metadata,
+      archived,
+    }
+    setLoadingArchive(true)
+
+    const uploadedMetadata = await uploadProjectMetadata(newMetadata)
+
+    if (!uploadedMetadata.success) {
+      setLoadingArchive(false)
+      return
+    }
+
+    setUriTx(
+      { cid: uploadedMetadata.cid },
+      {
+        onDone: () => setLoadingArchive(false),
+        onConfirmed: () => {
+          // Create github issue when archive is requested
+          // https://docs.github.com/en/rest/reference/issues#create-an-issue
+          octokit.request('POST /repos/jbx-protocol/juice-interface/issues', {
```

Comment:
**suggestion (non-blocking)** would be cool to add a github label although couldn't find docs for how to do that

Reply:
yeah love either. We can slap whatever we use on the Crowdin PRs too